types/mercadopago-sdk-js/package.json

```diff
@@ -11,10 +11,5 @@
     "devDependencies": {
         "@types/mercadopago-sdk-js": "workspace:."
     },
-    "owners": [
-        {
-            "name": "mercadopago",
```

Comment:
this doesn't seem right; github.com/mercadopago still exists and points to the company which presumably created these types.

Reply:
Probably someone converted a user into an org. Orgs can't be owners.